phpcs.xml.dist

```diff
@@ -39,13 +39,13 @@
 	<rule ref="WordPress.NamingConventions.PrefixAllGlobals">
 		<properties>
 			<!-- Value: replace the function, class, and variable prefixes used. Separate multiple prefixes with a comma. -->
-			<property name="prefixes" type="array" value="core_sitemaps"/>
+			<property name="prefixes" type="array" value="sitemaps"/>
 		</properties>
 	</rule>
 	<rule ref="WordPress.WP.I18n">
 		<properties>
 			<!-- Value: replace the text domain used. -->
-			<property name="text_domain" type="array" value="core-sitemaps"/>
+			<property name="text_domain" type="array" value="sitemaps"/>
```

Comment:
As mentioned, can't change the text domain

Reply:
👍